### DISCUSSION TITLE
Meeting List Download

### DISCUSSION THREAD

**Discussion by ATLCOTech** [code4recovery/pdf#62] [General]
Hi All - we are trying to download the Meeting List for Atlanta and am getting an server error 500. 

![image](https://github.com/code4recovery/12-step-meeting-list/assets/147879274/c667ed27-2fd5-406a-8ec5-c0f95185b464)

**Comment by anchovie91471** [comment]
Hey @ATLCOTech! This is an issue with our PDF service so I've transferred it over to the repository so we can track it. I am looking into the issue and will report back. Thank you for letting us know!

Anthony B

**Reply by anchovie91471** [reply]
@ATLCOTech, can you include a screenshot of the main screen with the settings you're using?

**Comment by SIA-WEBSUP** [comment]
I just ran the PDF generator on my feed with no issues...
    https://nassauintergroup.org/wp-admin/admin-ajax.php?action=meetings
    
When I ran it on your feed, it failed... likely because of a CDN/Cloudfare issue...
    https://atlantaa.org/wp-admin/admin-ajax.php?action=meetings
    
 There are other threads dealing with this... I'll find them and post here.

**Reply by SIA-WEBSUP** [reply]
This discussion thread on the TSML repo might be related...https://github.com/code4recovery/12-step-meeting-list/discussions/946

**Reply by SIA-WEBSUP** [reply]
Right now, invoking your feed redirects to http://ww1.atlantaa.org/ !!
I'm hoping that you're in the middle of investigating / fixing this  issue.

**Reply by anchovie91471** [reply]
@SIA-WEBSUP, I believe you have the domain wrong. It is atlantaaa.org. It appears you're missing an "a".

**Reply by SIA-WEBSUP** [reply]
@anchovie91471 , my bad... yes... I see their feed is working correctly... but is barfing in the PDF generator...

**Comment by ATLCOTech** [comment]
Do you mean from WP?

Sincerely,

Mark J
Tech & Newsletter Chair

If you have a newsletter submission, please send to ***@***.***
.
If you have a website comment, issue or would like to join the tech/web
committee email ***@***.*** ***@***.***>
To contact the steering committee email ***@***.***
Central Office ***@***.***

**Comment by anchovie91471** [comment]
The screen you see after clicking the "Generate PDF" button. Similar to this.


<img width="764" alt="Screenshot 2024-02-01 at 11 00 38 AM" src="https://github.com/code4recovery/pdf/assets/9162021/a812cbac-5588-48f5-8910-052acb86fff7">

**Comment by ATLCOTech** [comment]
Here ya go.

[image: image.png]
Sincerely,

Mark J
Tech & Newsletter Chair

If you have a newsletter submission, please send to ***@***.***
.
If you have a website comment, issue or would like to join the tech/web
committee email ***@***.*** ***@***.***>
To contact the steering committee email ***@***.***
Central Office ***@***.***

**Reply by anchovie91471** [reply]
@ATLCOTech  Ok, I think we've fixed the issue. Give it a try again.

**Comment by ATLCOTech** [comment]
Not sure what you're referring to here. But the problem was external for us
and was resolved. by Anthony B.

[image: image.png]
Sincerely,

Mark J
Tech & Newsletter Chair

If you have a newsletter submission, please send to ***@***.***
.
If you have a website comment, issue or would like to join the tech/web
committee email ***@***.*** ***@***.***>
To contact the steering committee email ***@***.***
Central Office ***@***.***
